changelog/changed-use-docsplay-for-ArmError.md

```diff
@@ -0,0 +1 @@
+Changed ArmError’s documentation to use docsplay::Display and avoid repetition
```

Comment:
As this change isn't visible to neither binary nor library users, I don't think it needs a changelog entry.